app/templates/sensor-button.html

```diff
@@ -0,0 +1,9 @@
+<span class="sensor-text-value">
+	<div ng-show="sensor.data.properties.value !== undefined">
+        <span ng-show="sensor.data.properties.value === false" class="balanced">Press</span>
```

Comment:
If these values reflect the state of the button, maybe Pressed/Released would be better choices.